src/media-groups.js

```diff
@@ -380,25 +376,9 @@ export const initialize = {
 
       // List of playlists that have an AUDIO attribute value matching the current
       // group ID
-      const groupPlaylists = playlists.filter(byGroupId(type, groupId));
 
       for (let variantLabel in mediaGroups[type][groupId]) {
         let properties = mediaGroups[type][groupId][variantLabel];
-
-        // List of playlists for the current group ID that have a matching uri with
-        // this alternate audio variant
-        const matchingPlaylists =
-          groupPlaylists.filter(byResolvedUri(properties.resolvedUri));
-
-        if (matchingPlaylists.length) {
-          // If there is a playlist that has the same uri as this audio variant, assume
-          // that the playlist is audio only. We delete the resolvedUri property here
-          // to prevent a playlist loader from being created so that we don't have
-          // both the main and audio segment loaders loading the same audio segments
-          // from the same playlist.
-          delete properties.resolvedUri;
```

Comment:
Removing this, can cause two segments to be download for the first segment of audio only videos. I think that we should log that as an issue to fix. This makes it so that we do not timeout when a video has separate audio

Reply:
So, the issue is that for an audio-only stream, the main segment loader and the audio segment loader would both download the audio segment. Previously, this `delete` statement would prevent the audio segment loader from being created. However, this causes issues when audio and video are unmuxed?

Reply:
Some regression testing should be done on playlists this affects as I believe there was another issue where one of the segment loaders were not calling end of stream

Reply:
Yes this causes issues with unmuxed audio/video. Yes we should do some regression testing, but I think that other `endofstream` fixes that have been done fixed those issues, at least for the streams that I have been testing with.